geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/AccumuloQueryPlan.scala

```diff
@@ -145,49 +168,55 @@ object AccumuloQueryPlan extends LazyLogging {
       }
     }
 
-    private def scanner(connector: Connector,
-                        table: String,
-                        auths: Authorizations): CloseableIterator[Entry[Key, Value]] = {
+    private def scanner(
+        connector: Connector,
+        table: String,
+        auths: Authorizations): CloseableIterator[Entry[Key, Value]] = {
       val scanner = connector.createBatchScanner(table, auths, numThreads)
-      scanner.setRanges(ranges)
+      scanner.setRanges(ranges.asJava)
       configure(scanner)
-      SelfClosingIterator(scanner.iterator, scanner.close())
+      SelfClosingIterator(scanner.iterator.asScala, scanner.close())
     }
   }
 
   // join on multiple tables - requires multiple scans
-  case class JoinPlan(filter: FilterStrategy,
-                      tables: Seq[String],
-                      ranges: Seq[org.apache.accumulo.core.data.Range],
-                      iterators: Seq[IteratorSetting],
-                      columnFamily: Option[Text],
-                      numThreads: Int,
-                      joinFunction: JoinFunction,
-                      joinQuery: BatchScanPlan) extends AccumuloQueryPlan {
+  case class JoinPlan(
```

Comment:
Do JoinPlans work in the distributed export?!?!

Reply:
No, it used the `AccumuloJobUtils.getSingleQueryPlan` method, which will disable join scans

Reply:
Is that documented?  (Do we care?)

Reply:
ah, not sure - it prints out some warnings i believe.